ci/sps-scripts/setup.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -eo pipefail
+set +u
+echo "===== setup.sh - start ====="
+
+# use CEL expression on trigger if commit should be skipped: header['x-github-event'] == 'push' && body.ref == 'refs/heads/ko-sps' && !body.head_commit.message.contains('[skip ci]')
+# see: https://cloud.ibm.com/docs/ContinuousDelivery?topic=ContinuousDelivery-tekton-pipelines&interface=ui#configure_triggering_events
+
+echo "Installing dependencies"
+source $WORKSPACE/$APP_REPO_FOLDER/installGolang.sh 1.24.4 amd64
```

Comment:
The version is no longer a parameter
```suggestion
source $WORKSPACE/$APP_REPO_FOLDER/installGolang.sh amd64
```